src/components/mobile-menu-container/mobile-menu-container.module.css

```diff
@@ -25,12 +25,22 @@
 	z-index: 2;
 }
 
-.mobileAuthenticationControls {
```

Comment:
Refactor here. Adjusted the wrap class to just provide the margin-bottom.

The flex styles only seemed to apply for the unauthenticated button controls, so I split those into their own class, which I also needed to do to adjust spacing with the theme switcher addition.